bin/create-test-app.js

```diff
@@ -80,11 +80,7 @@ program
         nodePackageManager = options.packageManager;
         break;
       case undefined:
-        if (options.install === "local") {
-          nodePackageManager = "pnpm";
-        } else if (options.install === "nightly") {
-          nodePackageManager = "npm";
-        }
+        nodePackageManager = "npm";
         break;
       default:
```

Comment:
We can just set npm in the default section and forget about undefined, right?

Reply:
If we don't add an undefined case then we can't omit the flag. Or did you have another fix in mind? 
I added `    let nodePackageManager = "npm";` and removed the undefined case.

Reply:
Maybe I'm missing something, but my idea was:

```
switch (options.packageManager) {
  case "yarn":
  case "pnpm":
    nodePackageManager = options.packageManager;
    break;
  default:
    nodePackageManager = "npm";
}
```

Reply:
But this is an internal script, I think it's fine and the code is simpler. But just an opinion.